modules/imaging_uploader/jsx/UploadForm.js

```diff
@@ -81,6 +81,34 @@ class UploadForm extends Component {
     }
 
     const fileName = data.mriFile.name;
+    // Make sure file is of type .zip|.tgz|.tar.gz format
+    const properExt = new RegExp('.(zip|tgz|tar.gz)$');
+    if (!fileName.match(properExt)) {
+      swal({
+        title: 'Invalid extension for the uploaded file!',
+        text: 'Filename extension does not match .zip, .tgz or .tar.gz ',
+        type: 'error',
+        confirmButtonText: 'OK',
+      });
+
+      let errorMessage = {
+        mriFile: 'The file ' + fileName + ' is not of type .tgz, .tar.gz or .zip.',
```

Comment:
```suggestion
        mriFile: 'The file ' + fileName + ' must be of type .tgz, .tar.gz or .zip.',
```

Not a big deal but I think this is a bit more instructive